core/utils/password_test.go

```diff
@@ -18,12 +19,12 @@ func TestVerifyPasswordComplexity(t *testing.T) {
 	}{
 		{"thispasswordislongenough", "", []error{}},
 		{"exactlyrightlen1", "", []error{}},
-		{"notlongenough", "", []error{utils.ErrPasswordMinLength}},
+		{"notlongenough", "", []error{errors.New("Password is 13 characters long")}},
```

Comment:
This error isn't actually helpful - it doesn't tell you what the requirements are

Reply:
The requirements are always part of the error message, as a header. See `ErrMsgHeader` and its assertion later on in this test case @samsondav 

What I'm basically doing is building a more readable error message similar to:
```
Must be longer than 12 characters
Must comprise at least 3 of:
    lowercase characters
    uppercase characters
    numbers
    symbols
Must not comprise:
    A user's API email
    More than three identical consecutive characters
```